cardano-lib/default.nix

```diff
@@ -252,6 +257,9 @@ let
       edgePort = 3001;
       networkConfig = import ./sanchonet-config.nix;
       usePeersFromLedgerAfterSlot = 21599922;
+      extraDbSyncConfig = {
+        enableConway = true;
```

Comment:
private chain, below, would also have enableConway = true.
I also do like the EnableFutureGenesis or similar generalization suggested in the PR header mentioned issue.

Reply:
Agreed with `EnableFutureGenesis`. Did the node config have or still has a similar option? It would be good to have consistency in names.